ding/data/buffer/middleware/priority.py

```diff
@@ -18,6 +22,18 @@ def __init__(
             IS_weight_power_factor: float = 0.4,
             IS_weight_anneal_train_iter: int = int(1e5),
     ) -> None:
+        """
+        Arguments:
+            - buffer (:obj:`Buffer`): The buffer to use PER.
+            - IS_weight (:obj:`bool`): Whether use importance sampling or not.
+            - priority_power_factor (:obj:`float`): The factor that adjust the sensitivity between\
+                the sampling probability and the priority level.
+            - IS_weight_power_factor (:obj:`float`): The factor that adjust the sensitivity between\
+                the sample rarity and sampling probability in importance sampling.
```

Comment:
maybe adding the greek symbols like beta in the article is better?

Reply:
Good suggestions! While the naming of those parameters needs to consider both simplicity and clarity, we may decide to rename it in the future.